app/api/article.tsx

```diff
@@ -131,7 +131,7 @@ class ArticleAPI extends PlutoAxios {
       params: {
         size,
         page,
-        query,
+        query: `text=${query}`,
```

Comment:
I think this isn't pretty and can make other people's mistake.
why don't you make the function or helper to make this format.